zap/src/main/java/org/zaproxy/zap/extension/search/SearchThread.java

```diff
@@ -279,6 +281,38 @@ private void search() {
                             }
                         }
                     }
+                    if (Type.Note.equals(reqType) && !pcc.allMatchesProcessed()) {
+                        String note = message.getNote();
+                        matcher = pattern.matcher(note);
+
+                        if (inverse && !pcc.allMatchesProcessed()) {
+                            if (!matcher.find()) {
+                                notifyMatchFound(currentRecordId, note, message, null, 0, 0);
+                            }
+                        } else {
+                            while (matcher.find() && !pcc.allMatchesProcessed()) {
+                                int note_extract_start =
```

Comment:
Should use camel case for variable names.

Reply:
Done in 5d1c62129.